python/lsst/fgcmcal/utilities.py

```diff
@@ -855,9 +855,15 @@ def extractReferenceMags(refStars, bands, filterMap):
 
 
 def lookupStaticCalibrations(datasetType, registry, quantumDataId, collections):
-    instrument = Instrument.fromName(quantumDataId["instrument"], registry)
-    unboundedCollection = instrument.makeUnboundedCalibrationRunName()
-
-    return registry.queryDatasets(datasetType,
-                                  dataId=quantumDataId,
-                                  collections=[unboundedCollection])
+    # For static calibrations, we search with a timespan that has unbounded
+    # begin and end; we'll get an error if there's more than one match (because
+    # then it's not static).
+    timespan = Timespan(begin=None, end=None)
```

Comment:
This will not be true with a "versioned camera," but as that's been a moving target for multiple years now, I'm happy to defer that issue until it becomes relevant.

Reply:
Yes, and I no longer think giving actual camera datasets different validity ranges is ever going to be a part of the versioned camera system.